.github/scripts/delete_stacks.sh

```diff
@@ -7,7 +7,8 @@ mapfile -t ACTIVE_STACKS_ARRAY <<< "$ACTIVE_STACKS"
 for i in "${ACTIVE_STACKS_ARRAY[@]}"
 do 
   echo "Checking if stack $i has open pull request"
-  PULL_REQUEST=${i//pr-/}
+  PULL_REQUEST=${i//pfp-pr-/}
+  PULL_REQUEST=${PULL_REQUEST//pr-}
```

Comment:
you will need to change the ACTIVE_STACKS command a few lines above as well to capture stacks named pfp-pr-<pull request id>